Development/Source/Swifter/MockNetworkServer.swift

```diff
@@ -183,6 +183,20 @@ private extension MockNetworkServer {
             fatalError("ATTENTION! " +
                 "The `\(String(describing: responseStub.response))` " +
                 "does not supported in the UI tests.")
+        case let .response(response):
+            guard let data = processHTTPBody(response.body) else {
+                return .internalServerError
+            }
+            return .raw(response.statusCode, response.reasonPhrase, response.headers, { try? $0.write(data) })
+        }
+    }
+
+    private func processHTTPBody(_ body: HttpBodyStub) -> Data? {
```

Comment:
Метод уже находится в `private extension`.
```suggestion
    func processHTTPBody(_ body: HttpBodyStub) -> Data? {
```